WordPress/src/main/java/org/wordpress/android/ui/activitylog/ActivityLogViewHolder.kt

```diff
@@ -0,0 +1,52 @@
+package org.wordpress.android.ui.activitylog
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.ProgressBar
+import android.widget.TextView
+import org.wordpress.android.R
+import org.wordpress.android.viewmodel.activitylog.ActivityLogListItemViewModel
+
+class ActivityLogViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
+        LayoutInflater.from(parent.context).inflate(R.layout.activity_log_list_item, parent, false)) {
+    private val summary: TextView = itemView.findViewById(R.id.action_summary)
+    private val text: TextView = itemView.findViewById(R.id.action_text)
+    private val thumbnail: ImageView = itemView.findViewById(R.id.action_icon)
+    private val progressBar: ProgressBar = itemView.findViewById(R.id.rewind_progress_bar)
+    private val button: ImageButton = itemView.findViewById(R.id.rewind_button)
+    private val header: TextView = itemView.findViewById(R.id.activity_header_text)
+
+    private lateinit var activity: ActivityLogListItemViewModel
+
+    init {
+        button.setOnClickListener {
+        }
+    }
+
+    fun bind(activity: ActivityLogListItemViewModel, previous: ActivityLogListItemViewModel?) {
+        this.activity = activity
+
+        summary.text = activity.summary
+        text.text = activity.text
+        header.text = activity.header
+
+        button.visibility = if (activity.isRewindable) View.VISIBLE else View.GONE
```

Comment:
this should just be `View.GONE` for all the fields since `rewind` is not implemented yet